atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -170,6 +172,24 @@ def validate_platforms_in_base_image(self, base_image):
                           'because platform descriptors are not defined')
             return
 
+        if '@sha256:' in str(base_image):
+            base_image_result = ImageName.parse(str(base_image))
+            base_image = base_image_result.copy()
+
+            try:
+                config_blob = get_config_from_registry(base_image, base_image.registry,
+                                                       base_image.tag,
+                                                       insecure=self.parent_registry_insecure)
+            except (HTTPError, RetryError, Timeout) as ex:
```

Comment:
But does this retry? I think it should, but I can't see what would drive that.

Reply:
get_config_from_registry is using RegistrySession, which has retries implemented as any other method in utils using that session